bootrom/sdboot/sd.c

```diff
@@ -162,7 +162,7 @@ static int copy(void)
 	dputs("CMD18");
 	kprintf("LOADING  ");
 
-	REG32(spi, SPI_REG_SCKDIV) = (F_CLK / 20000000UL);
+	REG32(spi, SPI_REG_SCKDIV) = (F_CLK / 16666666UL);
```

Comment:
Won't this break VC707 boot ?

Reply:
No, it doesn't. I tried it on the board. It'll be just slower to read the image from SD card.